src/components/ha-combo-box.ts

```diff
@@ -226,65 +225,73 @@ export class HaComboBox extends LitElement {
   }
 
   private _openedChanged(ev: ComboBoxLightOpenedChangedEvent) {
-    const opened = ev.detail.value;
-    // delay this so we can handle click event before setting _opened
-    setTimeout(() => {
-      this.opened = opened;
-    }, 0);
+    this.opened = ev.detail.value;
     // @ts-ignore
```

Comment:
I think we can remove it because observer handled with case.

Reply:
remove what?

Reply:
The setTimeout

Reply:
I don't think the `setTimeout` has anything todo with the observers, I believe it was for handling the `_toggleOpen ` case

Reply:
Ok, I will keep it

Reply:
Just tested, without the delay the combox will not close anymore with the button, maybe clarify the comment :-)